### ISSUE TITLE
getUserFeedContent Error

### ISSUE THREAD

**Issue by rafal-range** [knocklabs/knock-swift#6]
In my app I'm using 

```
knockFeedManager.getUserFeedContent(options: nil) { result in
    switch result {
        case let .success(feed): print(feed)
        case let .failure(error): print(error)
    }
}
```

This results in an error being logged:
```
valueNotFound(Swift.Array<Swift.String>, Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "entries", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0), CodingKeys(stringValue: "tenant", intValue: nil)], debugDescription: "Cannot get unkeyed decoding container -- found null value instead", underlyingError: nil))
```

It looks like the feed content has no tenant - which is expected as there's no tenant configured or specified in the options.

**Comment by matgar26** [comment]
@rafal-range Thanks for pointing this out. Ive finishes a fix for this in v 0.2.1. Please let me know if you see the issue persist

**Comment by rafal-range** [comment]
I can confirm it's fixed in v0.2.1, thanks for quick response
